vumi/persist/fake_redis.py

```diff
@@ -502,6 +502,9 @@ def ltrim(self, key, start, stop):
 
     @maybe_async
     def expire(self, key, seconds):
+        # raise ResponseError for non-int values to mimic Redis behaviour
+        if type(seconds) is not int:
```

Comment:
You need to also allow it to be type `long` (Python has two kinds of ints) and it might be better to do `if isinstance(seconds, (int, long))` to allow sub-classes of these types to also be allowed (it depends a bit what the real redis libraries do).